<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VICViewer</class>
 <widget class="QMainWindow" name="VICViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VIC Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="charsetFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>250</width>
      <height>250</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QFrame" name="matrixFrame">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>391</width>
      <height>251</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Internal</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>160</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>VC</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="VC"/>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Y</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>VMLI</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="VMLI"/>
      </item>
      <item row="4" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Y compare</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSpinBox" name="yCompare"/>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="VCBASE"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>X (cycle)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="y"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>RC</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>VCBASE</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="xCycle"/>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="RC"/>
      </item>
      <item row="5" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>y scroll</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>x scroll</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="xScroll"/>
      </item>
      <item row="5" column="3">
       <widget class="QSpinBox" name="yScroll"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>0</y>
      <width>181</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Colors</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="3" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>background 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="borderColor"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>background 0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>border</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>background 1</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>background 3</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="bgColor0"/>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="bgColor1"/>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="bgColor2"/>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="bgColor3"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>141</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Graphics Mode</string>
    </property>
    <widget class="QCheckBox" name="ECM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>extended color</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="BMM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>bitmap</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="MCM">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>131</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>multi color</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>190</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>Border</string>
    </property>
    <widget class="QCheckBox" name="CSEl">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>CSEL</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="RSEL">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>RSEL</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>0</y>
      <width>201</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Memory</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>247</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="bank15">
        <property name="text">
         <string>15: ~ CIA2 PA1</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="vm13">
        <property name="text">
         <string>13: VM13</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="cb12">
        <property name="text">
         <string>12: CB12</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>VM</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="cb11">
        <property name="text">
         <string>11: CB11</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="cb13">
        <property name="text">
         <string>13: CB13</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="vm10">
        <property name="text">
         <string>10: VM10</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="vm11">
        <property name="text">
         <string>11: VM11</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="bank14">
        <property name="text">
         <string>14: ~ CIA2 PA0</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QLineEdit" name="vmAddr">
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QCheckBox" name="vm12">
        <property name="text">
         <string>12: VM12</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_17">
        <property name="text">
         <string>CB</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QLineEdit" name="cbAddr">
        <property name="text">
         <string>0000</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
